CONTRIBUTION.md

```diff
@@ -0,0 +1,148 @@
+
+# New to Git
+
+Follow these resources: https://lab.github.com and https://try.github.com/
+
+# Style guide
+
+The Kubesimplify website is hosted in this repository and is built using [Docusaurus2.0](). Once a pull request has been submitted, a preview deployment will be built and made available to you and other contributors on your PR to review.
+
+# Contributing to Kubesimplify website
+
+Please follow these steps and note these guidelines to begin contributing:
+
+1. The first step is to set up the local development environment. See [this](#how-to-contribute) on how to do the same.
+1. Take a look at the existing [Issues](https://github.com/kubesimplify/website/issues) or [create a new issue!](https://github.com/kubesimplify/website/issues/new/choose)
+1. A good way to easily start contributing is to pick and work on a [good first issue](https://github.com/kubesimplify/website/labels/good%20first%20issue) or [help wanted](https://github.com/kubesimplify/website/labels/help%20wanted). We try to make these issues as clear as possible and provide basic info on how the code should be changed, and if something is unclear feel free to ask for more information on the issue.
+1. Add screenshots or screen captures to your Pull Request to help us understand the effects of the changes proposed in your PR.
+
+# How to Contribute
+
+## Prerequisites
+
+Make sure you have the following prerequisites installed on your operating system before you start contributing:
+
+- [Nodejs](https://nodejs.org/en/) version 16.14 or above.
+
+  To verify run:
+
+  ```
+  node -v
+  ```
+
+- [Yarn](https://yarnpkg.com) version 1.2 or above.
+
+  To verify run:
+
+  ```
+  yarn -v
+  ```
+
+## Set up your Local Development Environment
+
+### This website is built using [Docusaurus2.0](https://docusaurus.io/), a modern static website generator.
+
+Follow the following instructions to start contributing.
+
+**1.** Fork [this](https://github.com/kubesimplify/website) repository.
+
+**2.** Clone your forked copy of the project.
+
+```
+git clone https://github.com/<your-github-username>/kubesimplify.git
+```
+
+**3.** Navigate to the project directory.
+
+```
+cd kubesimplify
+```
+
+**4.** Add a reference (remote) to the original repository.
+
+```
+git remote add upstream https://github.com/kubesimplify/website.git
+```
+
+**5.** Check the remotes for this repository.
+
+```
+git remote -v
+```
+
+**6.** Create a new branch.
+
+```
+git checkout -b <your_branch_name>
+```
+
+**7.** Install the dependencies for running the site.
+
+```
+yarn
+```
+
+**8.** Run the site locally to preview changes.
+
+```
+yarn start
+```
+
+This will run a local webserver with "live reload" conveniently enabled at `localhost/3000`.
+
+**9.** Make desired changes to files.
+
+**10.** Track your changes.
+
+```
+git add .
```

Comment:
```suggestion
git add  <file_name>
```

Reply:
This is absolutely important! Thanks for making this change @AvineshTripathi